### ISSUE TITLE
프로젝트 제안서

### ISSUE THREAD

**Issue by Seeun-Lim** [SEEUNL/Recycle#2]
1. 주제 선정의 이유
실생활에 보다 직접적으로 도움이 될 수 있는 주제를 선정하고 싶었고 분리수거가 아직 사람에 의해 일일이 수행된다는 점에서 분류해주는 기계가 있다면 다양한 기대효과를 창출할 수 있을 것 같아 선정하게 되었음. 

2. 기존 연구/ 개발 조사 결과
 1 - POSTECH 기계공학과의 금속감지기와 Matlab 이용한 자동 분리수거 장치
   →컴퓨터 자체적으로 학습 불가
 2 - Recycle AI의 Tensorflow와 라즈베리파이3를 이용한 자동 분리수거 장치
   →쓰레기를 '재활용쓰레기', '매립 쓰레기', '퇴비 쓰레기'로 분류
 3 - 수퍼빈의 네프론
   →캔/페트병 분리수거 후 현금 적립
 Image Based Trash Classication using Machine Learning Algorithms
for Recyclability Status by Mandar Satvilkar(2018)에 따르면 재활용품 분류에 있어서다양한 머신러닝 알고리즘중 CNN이 다른 알고리즘에 비해 좋은 성능을 보여줌.

3. 프로젝트에 사용할 데이터 셋
Train set: github.com/garythung/trashnet(유리,종이,판지,플라스틱,고철,쓰레기로 분류)
 	  www.image-net.org/index 
          클래스 당 500개 이미지를 모아 초기 학습데이터로 이용할 것임.
Test set: www.flickr.com
	www.image-net.org/index
	 이미지 인식 학습에 주로 이용되는 사이트들이다. 위 사이트의 이미지를 활용하여 
	 분리수거물에 대한 인공지능 인식률을 시험해 볼 것임.

4. 역할 분담
임세은 | Open CV로 이미지 객체 분리, PPT 담당
김남규 | 데이터셋 다운로드 및 변환, 발표 담당
김성빈 | 딥러닝 모델 학습, 발표 담당
임지윤 | 딥러닝 모델 평가, PPT 담당

5. 프로젝트를 통한 예상 결과물
1) 재활용품 실제 물체나 종이에 인쇄된 물체를 카메라에 인식(추후에 라즈베리 파이 사용  할 수도 있음)
2) 프로그램의 이미지 인식(객체 분리 방식) 후 저장
3) 객체를 분류 인식하여 화면에 표시

6. 기대효과
1) 국가 차원의 기대효과 >> 기계라는 좀 더 정확한 방법을 이용함으로써 재활용률을 높여 국가의 환경보호와 발전에 이바지함
2) 개인 차원의 기대효과 >> 사람에게 번거롭고 애매한 재활용품들을 기계가 대신 분류해줌으로써 더 쉽게 재활용 분류가 가능함
